spec/features/user_signs_in_spec.rb

```diff
@@ -5,12 +5,12 @@
   I want to sign in
   So that I can regain access to my account
 } do
-  scenario 'specify valid credentials' do
+  scenario 'specify valid username' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.